uptime-kuma/Chart.yaml

```
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.0.10
+version: 1.0.11
```

```suggestion
version: 1.1.0
```

would bundle this with the service monitor PR and tag is as 1.1.0 ;)